eng/scripts/Update-VcpkgBeta.ps1

```diff
@@ -77,9 +77,15 @@ try {
     $rawVcpkgConfig = Get-Content -Raw -Path $vcpkgConfigPath
     $vcpkgConfig = ConvertFrom-Json $rawVcpkgConfig
 
+    $azSdkRepoBaseline = (Get-Content $RepoRoot/vcpkg.json -Raw -ErrorAction SilentlyContinue | ConvertFrom-Json).'builtin-baseline'
```

Comment:
`$RepoRoot` is not a param to this script. 

This should probably be `$PSScriptRoot/../../` or something. This will make the path relative to `eng/scripts/Update-VcpkgBeta.ps1`

Reply:
Thank you! Fixed in the new iteration.